### ISSUE TITLE
Don't add hash to external sources

### ISSUE THREAD

**Issue by richarddavenport** [posthtml/posthtml-hash#22]
If you have links to external sources like this:

```
<link
      rel="stylesheet"
      href="https://fonts.googleapis.com/css?family=Open+Sans:400,400i,600,600i,700,700i"
    />
```
The plugin throws an error trying to find the file.

Is there a way around this?

**Comment by metonym** [comment]
This is definitely unexpected behavior. Will add the behavior to ignore external URLs.

**Comment by richarddavenport** [comment]
@metonym Thanks! Appreciate your work!

**Comment by metonym** [comment]
Released in version 0.2.3

Thanks @richarddavenport for raising this issue!
